SIGNALESP.ino

```diff
@@ -2,6 +2,7 @@
 #include "compile_config.h"
 
 #define PROGNAME               " SIGNALESP "
+#define PROGVERS               "3.4.0-dev _200113"
```

Comment:
wie gesagt, das wird schon in SIGNALduino.ino definiert und die arduino ide kopiert alle ino dateien einfach zusammen.

PS: ich habe die ino dateien in .h dateien migriert.

Reply:
also gehst du von der Struktur nun ab und ich muss die neue nehmen ?

Reply:
Weiss ich noch nicht, ich bin am probieren, aber warum denkst Du, dass es 2x definiert werden muss? Es ist doch auch in diesem PR 2x definiert

Die für plattformio angepasste Version habe ich noch nicht mit der arduino ide compiliert. Ich habe mir nur überlegt, dass es recht einfach gehen müsste

Reply:
Ohne diese Zeile bringt arduino ide ein Fehler. Ich habe es soeben getestet, es geht nur, wenn es ebenso dort nochmal definiert wurde wie im PR.

Wenn du es nicht machst, schlägt es immer fehl hier
`			MSG_PRINT(F("V " PROGVERS PROGNAME));`
mit

```
exit status 1
expected ')' before 'PROGVERS'

```

Reply:
https://github.com/RFD-FHEM/SIGNALDuino/blob/dev-r3.4/SIGNALDuino.ino

Compilierst Du denn auch diese Datei?

Reply:
Für den ESP nutze ich die SIGNALESP.ino.
Nun sage nicht, das es über die andere geht *grrr* (dann hau wech diese wenn es geht ;-) )

Reply:
Mit der neuen Struktur gibt es solche überraschungen nicht mehr, da die einzige ino. datei aufs nötigste reduziert wurde `#include "src/main.cpp"` :)

Reply:
Ich habe nun beide Dateien in das Verzeichnis kopiert und lade die SIGNALduino.ino, er läd die SIGNALESP.ino aber die Variable ist noch nötig dafür im dev-r3.4 Branch, NICHT dein neuer Branch.

Reply:
Das Projekt nennt sich SIGNALduino und die Arduino ide versucht immer eine signalduino.ino zu laden. Alle anderen ino dateien werden mit dieser zusammen zu einer header Datei kopiert (CPP kennt keine .ino Dateien)

Kurzum. Ja, die Basis war schon immer die signalduino.ino und die signalesp.ino wurde nachgeladen.

Reply:
Das wahre ist es noch nicht.

Wenn man die Ino aufruft, so sieht man die compile_config nicht und selbst dann schlägt das kompilieren fehlt :-(

Reply:
Dank @maxgerhardt ist auf jedenfall die Funktion gegeben mit der Anpassung.